libraries/NetworkUtils.py

```diff
@@ -5,13 +5,17 @@
 from libraries.provision.ansible_runner import AnsibleRunner
 
 from keywords.exceptions import ProvisioningError
+from keywords.utils import log_info
 
 
 class NetworkUtils:
 
     def list_connections(self):
-        output = subprocess.check_output("netstat -ant | awk '{print $6}' | sort | uniq -c | sort -n", shell=True)
-        print(output)
+        log_info("\nSocket usage on mobile-testkit client ...")
```

Comment:
It results in better formatting in the the test output when running. We can take it out. It is just harder to spot then. What do you think?